frontend/cypress/fixtures/users.json

```diff
@@ -1,10 +1,10 @@
 [
   {
     "id": 1,
-    "name": "Admin McAwesome",
-    "username": "admin@test.com",
-    "email": "admin@test.com",
-    "password": "Spp9k6i7",
+    "name": "Admin",
+    "username": "Admin",
+    "email": "admin@mail.com",
+    "password": "atwookimomo@23",
```

Comment:
This is going to break all tests on deployment ... instead of changing these to suit yours, why not create a user with the expected credentials. I mean, this didn't need to change at all really.

Reply:
I was getting an unauthorized error

Reply:
I'm sure ... so create a user with the expected credentials, either via Django admin or with `[createsuperuser](https://djangocentral.com/creating-super-user-in-django/)` ... u only need the matching username & password.